docs/create/examples.md

```diff
@@ -0,0 +1,33 @@
+---
+id: examples
+title: Examples
+---
+
+Before diving into creating your own XRPackages, it can be useful to see some examples and play around with the code!
+
+If you think something is missing from this list, you can submit a pull request on <a href="https://github.com/webaverse/docs" target="_blank" rel="noopener noreferrer">GitHub</a>.
+
+# GLB XRPKs
+- [Parkour](https://github.com/avaer/parkour) - A big industrial land scene.
+- [Home](https://github.com/avaer/home) - A home environment scene.
+- [Kokiri Forest](https://github.com/avaer/kokiri-forest) - A Kokiri Forest scene.
+- [Glorb](https://github.com/avaer/glorb) - A wearable model.
+- [Tip Jar](https://github.com/avaer/tipjar) - A tip jar model.
+
+# ThreeJS Website XRPKs
+- [Comet](https://github.com/avaer/comet) - A comet.
+- [Land](https://github.com/avaer/land) - A colorful geometric land.
+- [Planet](https://github.com/avaer/planet) - A big geometric planet.
+- [Voxels](https://github.com/avaer/voxels) - A collection of voxels.
+- [Physics Cube](https://github.com/avaer/physicscube) - A cube with physics.
+- [Portal](https://github.com/avaer/portal) - A set of portals.
+- [Lightsaber](https://github.com/avaer/lightsaber) - A lightsaber.
+- [Mirror](https://github.com/avaer/mirror) - A mirror.
+- [Sheild](https://github.com/avaer/shield) - A shield.
+- [Weapons](https://github.com/avaer/weapons) - A table of weapons.
+
+# VRM XRPKs
+- [Hatsune Miku](https://github.com/avaer/miku) - A Hatsune Miku avatar.
+
+# Tools
+- [XRPackage Builder](https://github.com/webaverse/xrpackage-build-action) - A GitHub action to automatically build your XRPackages.
```

Comment:
I think this would be outdated for the current XRPK format. We could just get rid of it from the docs until we can test that it's working well with the current packages.

https://github.com/webaverse/app/issues/270